backend/test/bookclub.js

```diff
@@ -0,0 +1,91 @@
+var should = require('should');
+var assert = require('assert');
+var request = require('supertest');
+var mongoose = require('mongoose');
+var User = require('../models/user.js');
+var Books = require('../models/book.js');
+
+describe('Routing', function() {
+  var url = 'localhost:8081';
+  // all the operations that are needed to setup the tests.
+  before(function(done) {
+    User.remove({}, function error (err) {
+    });
+    user1 = new User ({
+      email: "supermimi@gmail.com",
+      username: "mimi",
+      password: "$2a$10$/jR0tK67zTM0xUYg6vjhDOBN2iqfRjSW5vHDk0czhfwwrErfRbcwm",
+      firstName: "Mohamed",
+      lastName: "El Sayed",
+      age: 18,
+      address: "Shobra",
+      profilePhoto: "default_avatar.png",
+      gender: "male",
+      toBeRead: [],
+      bookClubs: []
+    });
+    user1.save();
+
+    user2 = new User ({
+      email: "superH@gmail.com",
+      username: "Hesham",
+      password: "$2a$10$/jR0tK67zTM0xUYg6vjhDOBN2iqfRjSW5vHDk0czhfwwrErfRbcwm",
+      firstName: "Hesham",
+      lastName: "El Ot",
+      age: 16,
+      address: "Shobra",
+      profilePhoto: "default_avatar.png",
+      gender: "male",
+      toBeRead: []
+    });
+    user2.save();
+
+    books = [];
+    done();
```

Comment:
You need to document like the following example
**/ @function editProfile - Called on GET "/api/users/:id"
* edit user info in the database and save
* @param {Object} req - Http request
* @param {Object} res - Http response
* @param {Object} next - Next middleware
*/